webpack.config.js

```diff
@@ -25,5 +29,15 @@ module.exports = {
       exclude: /(node_modules)/,
       loader: 'babel-loader'
     }]
-  }
+  },
+
+  plugins: [
+    new webpack.DefinePlugin({
+      SPEC_FILE_FILTER: process.env.SPEC_FILE_FILTER || defaultSpecFileFilter
+    }),
+
+    new webpack.DefinePlugin({
+      DEFAULT_SPEC_FILE_FILTER: defaultSpecFileFilter
+    })
```

Comment:
do you have to have two separate DefinePlugin's here? versus:

```
new webpack.DefinePlugin({
  SPEC_FILE_FILTER: process.env.SPEC_FILE_FILTER || defaultSpecFileFilter,
  DEFAULT_SPEC_FILE_FILTER: defaultSpecFileFilter
})
```

Reply:
That works as well. I'll make this change.